tests/test_utils.py

```diff
@@ -25,3 +27,24 @@ def test_non_empty_keys(self):
         clean_data = airbrake.utils.non_empty_keys(data)
 
         self.assertEqual(expected_data, clean_data)
+
+    def test_get_local_git_revision(self):
+        rev = airbrake.utils.get_local_git_revision()
+        self.assertIsNotNone(rev)
+
+        has_fs_access = None
+        head_ref_path_file = os.path.join(airbrake.utils._get_git_path(),
+                                          "HEAD")
+        with open(head_ref_path_file, 'r') as test_file:
+            ref_path = test_file.read().strip()
+            if ref_path:
+                has_fs_access = True
+
+        if has_fs_access:
+            rev_file = airbrake.utils._git_revision_from_file()
+            self.assertIsNotNone(rev_file)
+
+        rev = subprocess.check_output(["git", "rev-parse", "HEAD"])
+        if rev:
+            rev_binary = airbrake.utils._git_revision_with_binary()
+            self.assertIsNotNone(rev_binary)
```

Comment:
Great idea! I have something like 4 commits above where I was trying to finagle a test that does that. Something is different when this runs in a docker environment, but I felt a bit pressed for time so I didn't run it down.